lib/influxdb/query/core.rb

```diff
@@ -122,7 +122,11 @@ def generate_payload(data)
       end
 
       def execute(query, options = {})
-        url = full_url("/query", q: query)
+        url = if options[:db]
```

Comment:
Thanks for reviewing my PR!

Since I made a modification, please check it out again.